lib/Util.php

```diff
@@ -10,76 +10,89 @@
 class Util {
 
```

Comment:
This file had several unused methods that were carried over from our boilerplate. Removed those while in this file.